internal/restic/node.go

```diff
@@ -297,11 +298,28 @@ func (node Node) writeNodeContent(ctx context.Context, repo Repository, f *os.Fi
 }
 
 func (node Node) createSymlinkAt(path string) error {
+
+	linkTarget := node.LinkTarget
+
 	// Windows does not allow non-admins to create soft links.
 	if runtime.GOOS == "windows" {
-		return nil
+		if !IsRunningAsAdminOnWindows {
+			debug.Log(
+				"can't restore symlink to %v without admin privileges on windows",
+				path,
+			)
+			return nil
+		}
+
+		// fix path so a restore to a different location will produce
+		// consistent data
```

Comment:
Is this necessary? It introduces an incompatbility between Windows and Unix.

Reply:
For me it was necessary to restore the symlinks properly, otherwise they would point to invalid locations. This might be related to https://github.com/restic/restic/issues/2004. If this one is done I hope to investigate https://github.com/restic/restic/issues/2004.

Reply:
On Unix, restore may produce invalid symlinks as well, but I guess the drive letters make this more painful on Windows. I'm not sure what would be the proper behavior here.

Reply:
> Is this necessary? It introduces an incompatbility between Windows and Unix.

Yes - When doing full system resore of Windows it needs the symbolic links to be restore - else user profiles will not be usable.